src/int-range.js

```diff
@@ -2,9 +2,9 @@ import { defaultEngine, generateCurry, randIntRange } from './utils.js'
 
 /**
  * Returns a random number within the integer range
- * @param min Minimum integer
- * @param max Maximum integer
- * @param engine PRNG of choice
+ * @param {number} min Minimum integer
```

Comment:
I'm unfamiliar with jsDoc, but I found the documentation here: https://jsdoc.app/tags-type.html

I'm curious, does this play well with TypeScript applications?

Reply:
Yep, I just want to push you a bit into the right direction here, once you did the rest of JSDoc's, you can generate `aimless.d.ts` (which is what goes later into the npm package)

I just added the command for that already, if you want to see:

![image](https://github.com/ChrisCavs/aimless.js/assets/5236548/8c29c3da-1ff3-40e1-b784-7b153e6faa73)

Reply:
i see, so i need typescript as a dev dependency.  once I run the command and generate the types into the types folder, will they automatically be bundled into `dist` on build?

Reply:
i have the type files generated, and i added a rollup step to bundle them, but it's not working.  not sure how to get these types declarations actually in the dist folder